### PR TITLE
Features/add stock market

### PR DESCRIPTION
**Pull request by jindra12**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "react/button-has-type": 0,
     "import/prefer-default-export": 0,
     "react/require-default-props": 0,
+    "prefer-promise-reject-errors": 0,
```
Comment (DorianSternVukotic) [inline .eslintrc:11]: why

File: package.json [hunk 2]
```diff
@@ -36,6 +36,7 @@
     "lodash": "^4.17.15",
     "markdown-to-jsx": "^7.4.5",
     "node-sass": "^9.0.0",
+    "object-hash": "^3.0.0",
```
Comment (DorianSternVukotic) [inline package.json]: i dont see this used in code?

File: src/components/InputComponents/InputSearchAddressName.js [hunk 3]
```diff
@@ -1,112 +1,61 @@
...
-    isRequired,
-    validate,
-    id,
-  };
-  const [inputValue, setInputValue] = useState(defaultValue || '');
+function InputSearch(props) {
+  const [loading, setLoading] = useState();
+  const [error, setError] = useState();
   const [suggestions, setSuggestions] = useState([]);
-  const [isSuggestedListShown, setIsSuggestedListShown] = useState(true);
```
Comment (DorianSternVukotic) [inline src/components/InputComponents/InputSearchAddressName.js:32]: why refactor identity autocomplete?

File: package.json (hunk 2 above)
Comment (jindra12) [reply package.json]: Removed

File: src/components/InputComponents/InputSearchAddressName.js (hunk 3 above)
Comment (jindra12) [reply src/components/InputComponents/InputSearchAddressName.js:32]: The component was opinionated towards RHF. Added back the obsolete component back which I will slowly eradicate through refactoring using ANTD

File: .eslintrc (hunk 1 above)
Comment (jindra12) [reply .eslintrc:11]: ANTD validation expects you to reject invalid objects as Promise.reject(string), doesn't make sense to wrap everything in error objects

File: src/components/Modals/AddLiquidityModal.js [hunk 4]
```diff
@@ -120,119 +126,166 @@ function AddLiquidityModal({
...
+  const amount1Desired = Form.useWatch('amount1Desired', form);
+  const amount2Desired = Form.useWatch('amount2Desired', form);
+  const details = Form.useWatch('details', form);
+
+  useEffect(() => {
+    if (asset1Focused && amount1Desired) {
+      handleChangeInput(amount1Desired, true);
+    }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [amount1Desired]);
```
Comment (SzymonZiebaNeti) [inline src/components/Modals/AddLiquidityModal.js:146]: Are you sure, you want to skip in dependecies asset1Focused?

File: src/components/Modals/AddLiquidityModal.js [hunk 5]
```diff
@@ -120,119 +126,166 @@ function AddLiquidityModal({
...
+
+  useEffect(() => {
+    if (asset2Focused && amount2Desired) {
+      handleChangeInput(amount2Desired, false);
+    }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [amount2Desired]);
```
Comment (SzymonZiebaNeti) [inline src/components/Modals/AddLiquidityModal.js:153]: Same with asset2Focused?

File: src/components/Modals/AddLiquidityModal.js (hunk 4 above)
Comment (jindra12) [reply src/components/Modals/AddLiquidityModal.js:146]: Yes, that is on purpose, this prevents additional useless rerenders

**Pull request merged**